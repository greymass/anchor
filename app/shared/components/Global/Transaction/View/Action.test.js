import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import GlobalTransactionViewAction from './Action';

const mockProps = {
  action: {},
};

describe('GlobalTransactionViewAction', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<GlobalTransactionViewAction {...mockProps} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});

