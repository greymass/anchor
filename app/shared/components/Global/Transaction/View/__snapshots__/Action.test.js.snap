// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalTransactionViewAction renders correctly 1`] = `
<Fragment>
  <Label
    color="grey"
    ribbon={true}
    size="large"
  >
    <Icon
      as="i"
      name="clipboard outline"
    />
    <LabelDetail />
  </Label>
  <t
    collapseStringsAfterLength={32}
    collapsed={false}
    defaultValue={null}
    displayDataTypes={false}
    displayObjectSize={false}
    enableClipboard={true}
    groupArraysAfterLength={100}
    iconStyle="square"
    indentWidth={4}
    name={null}
    onAdd={false}
    onDelete={false}
    onEdit={false}
    onSelect={false}
    shouldCollapse={false}
    sortKeys={false}
    src={Object {}}
    style={
      Object {
        "padding": "1em",
      }
    }
    theme="harmonic"
    validationMessage="Validation Error"
  />
</Fragment>
`;
